### PR TITLE
Update audio editor UI

### PR DESCRIPTION
**Pull request by Alvareon**
Updated Audio Editor UI according to the Figma:

https://github.com/WEKIT-ECS/MIRAGE-XR/assets/34060044/d5c339e7-76bb-4acf-863b-6f74a20ada54

### CODE REVIEW

**Review by fwild — CHANGES_REQUESTED** [review]
Looks good to me, just noticed the use of "UnityEngine.Debug.LogError("[111] Initialization");" at the top of the file (line 82) - would be better to use "[AudioAugmentation]" in there, and probably just Debug.LogWarning or Debug.LogTrace, as otherwise we will get a lot of notifications in Sentry!

File: Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs [hunk 1]
```diff
@@ -23,7 +24,7 @@ public class IntHolder : ObjectHolder<int> { }
     public override ContentType editorForType => ContentType.AUDIO;
 
     [SerializeField] private Button _btnAudioSettings;
-    [SerializeField] private Button _btnCancel;
+    [SerializeField] private Button _btnMicRecording1, _btnMicRecording2;
```
Comment (cawke) [inline Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs]:
```suggestion
    [SerializeField] private Button _btnMicRecording1; 
    [SerializeField] private Button _btnMicRecording2;
```

Is it possible to give them a more specific name without numbers?

File: Assets/MirageXR/Design/UI_Elements/PNG/Audio/recording_new.png.meta [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  streamingMipmaps: 0
+  streamingMipmapsPriority: 0
+  vTOnly: 0
+  ignoreMasterTextureLimit: 0
+  grayScaleToAlpha: 0
+  generateCubemap: 6
+  cubemapConvolution: 0
+  seamlessCubemap: 0
+  textureFormat: 1
+  maxTextureSize: 2048
```
Comment (cawke) [inline Assets/MirageXR/Design/UI_Elements/PNG/Audio/recording_new.png.meta]: change the maximum size for all sprites to 256

File: Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/NewUI/AudioEditor_v2.prefab [hunk 3]
```diff
@@ -895,115 +1088,606 @@ MonoBehaviour:
...
+  m_EditorClassIdentifier: 
+  m_Material: {fileID: 0}
+  m_Color: {r: 1, g: 1, b: 1, a: 1}
+  m_RaycastTarget: 1
+  m_RaycastPadding: {x: 0, y: 0, z: 0, w: 0}
+  m_Maskable: 1
+  m_OnCullStateChanged:
+    m_PersistentCalls:
+      m_Calls: []
+  m_text: Press adain to stop recording
```
Comment (cawke) [inline Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/NewUI/AudioEditor_v2.prefab]: image

again

**Review by cawke — CHANGES_REQUESTED** [review]

File: Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/NewUI/AudioEditor_v2.prefab (hunk 3 above)
Comment (Alvareon) [reply Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/NewUI/AudioEditor_v2.prefab]: fixed

File: Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs (hunk 1 above)
Comment (Alvareon) [reply Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs]: fixed

**Comment by Alvareon** [comment]
> Looks good to me, just noticed the use of "UnityEngine.Debug.LogError("[111] Initialization");" at the top of the file (line 82) - would be better to use "[AudioAugmentation]" in there, and probably just Debug.LogWarning or Debug.LogTrace, as otherwise we will get a lot of notifications in Sentry!

I deleted it. It was only needed for debugging.

File: Assets/MirageXR/Design/UI_Elements/PNG/Audio/recording_new.png.meta (hunk 2 above)
Comment (Alvareon) [reply Assets/MirageXR/Design/UI_Elements/PNG/Audio/recording_new.png.meta]: done

File: Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs [hunk 4]
```diff
@@ -23,7 +24,7 @@ public class IntHolder : ObjectHolder<int> { }
     public override ContentType editorForType => ContentType.AUDIO;
 
     [SerializeField] private Button _btnAudioSettings;
-    [SerializeField] private Button _btnCancel;
+    [SerializeField] private Button _btnMicRecording, _btnMicReRecording;
```
Comment (cawke) [inline Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs]:
```suggestion
    [SerializeField] private Button _btnMicRecording;
    [SerializeField] private Button _btnMicReRecording;
```

**Review by cawke — CHANGES_REQUESTED** [review]

**Review by cawke — APPROVED** [review]

**Pull request merged**